lib/browser/api/app.js

```diff
@@ -45,6 +45,13 @@ if (process.platform === 'darwin') {
   }
 }
 
+if (process.platform === 'linux' && bindings.unityLauncherAvailable()) {
```

Comment:
The existence of API should not depend on current environment, if an API is available on one Linux distribution, it should be able to be called on other Linux distributions, even though it may fail.

We can add a `app.unityLauncher.isAvailable` API to let user decide how to behave on different platforms.